metricbeat/module/kafka/partition/partition.go

```diff
@@ -109,25 +111,31 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 		}
 
 		for _, partition := range topic.Partitions {
-			// partition offsets can be queried from leader only
-			if broker.ID() != partition.Leader {
-				debugf("broker is not leader (broker=%v, leader=%v)", broker.ID(), partition.Leader)
-				continue
-			}
-
 			// collect offsets for all replicas
-			for _, id := range partition.Replicas {
-
-				// Get oldest and newest available offsets
-				offOldest, offNewest, offOK, err := queryOffsetRange(broker, id, topic.Name, partition.ID)
```

Comment:
`queryOffsetRange` can be removed, I think it is not used in any other place.

Reply:
Removed.